grpc-sys/build.rs

```diff
@@ -333,12 +333,15 @@ fn bindgen_grpc(mut config: bindgen::Builder, file_path: &PathBuf) {
 fn config_binding_path(config: bindgen::Builder) {
     let file_path: PathBuf;
     let target = env::var("TARGET").unwrap();
-    println!("cargo:rerun-if-changed=bindings/{}-bindings.rs", &target);
     match target.as_str() {
         "x86_64-unknown-linux-gnu" | "aarch64-unknown-linux-gnu" => {
             file_path = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap())
                 .join("bindings")
                 .join(format!("{}-bindings.rs", &target));
+            println!(
+                "cargo:rerun-if-changed=bindings/{}",
+                file_path.as_path().as_os_str().to_str().unwrap()
```

Comment:
I suggest to use relative path. Otherwise sharing building cache can become a problem.